Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -15,21 +18,52 @@ public Inventory()
 
         public Item[] GetItems(string name)
         {
-            return new Item[0];
+            var foundItems = new Item[0];
+            foreach (var item in _items)
+            {
+                if (item.GetName() == name)
+                {
+                    foundItems = item.AddItem(foundItems,item); 
+                }
+            }
+
+            return foundItems;
         }
 
         public void AddItem(Item item)
         {
+            _items = item.AddItem(_items, item);
```

Comment:
You add item to item? Something isn't right, please move the function outside of item.